SharpBucketCli/Program.cs

```diff
@@ -85,8 +85,8 @@ private void ListenToInteractiveCommands()
 
         private void UseEnvironmentCredentials()
         {
-            var consumerKey = Environment.GetEnvironmentVariable("SB_CONSUMER_KEY");
-            var consumerKeySecret = Environment.GetEnvironmentVariable("SB_CONSUMER_SECRET_KEY");
+            var consumerKey = System.Environment.GetEnvironmentVariable("SB_CONSUMER_KEY");
+            var consumerKeySecret = System.Environment.GetEnvironmentVariable("SB_CONSUMER_SECRET_KEY");
```

Comment:
Why those changes ?

Reply:
Ok for this ne I saw that it's due to a name conflict with the new Environement class from the new endpoints!